fall18/p1/main.py

```diff
@@ -0,0 +1,199 @@
+# This line is a comment because it starts with a pound sign (#). That
+# means Python ignores it.  A comment is just for a human reading the
+# code.  This project involves 20 small problems to give you practice
+# with operators, types, and boolean logic.  We'll give you directions
+# (as comments) on what to do for each problem.
+
+############################################################
+print('Problem 1')
+
+# we did this one for you!
+
+print('one plus one is:')
+print(1+1)
+print()
+
+############################################################
+print('Problem 2')
+
+print('two plus two is:')
+
+# now fix the next line so it prints 4 by adding 2 with 2.  If you just
+# print 4, you may pass the tests and initially get a good grade
+# automatically, but we'll take away those points later when a TA
+# reads your solution!  (this example about grading applies
+# generally).
+print(0)
+
+print()
+
+############################################################
+print('Problem 3')
+
+# we did this one for you too!
+
+print('the type of 5 is:')
+print(type(5))
+print()
+
+############################################################
+print('Problem 4')
+
+print('the type of 1.5 is:')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 5')
+
+print('the type of 5.0 is:')
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 6')
+
+print('the type of "5.0" is:') # note the quotes!
+
+# put something in the parenthesis here to get an answer
+print()
+print()
+
+############################################################
+print('Problem 7')
+
+print('the type "True" is:')
```

Comment:
Change this as 'the type of True is: ' ("of" is missing which leads to test failing).
Also, adding a comment saying "# note the quotes!" here too might be helpful.